frontend/src/components/dashboard/aliases/Alias.tsx

```diff
@@ -8,8 +8,7 @@ import {
   useTooltipTrigger,
 } from "react-aria";
 import styles from "./Alias.module.scss";
-import copyIcon from "../../../../../static/images/copy-to-clipboard.svg";
-import arrowDownIcon from "../../../../../static/images/arrowhead.svg";
+import { ArrowDownIcon, CopyIcon } from "../../Icons";
```

Comment:
These are actually React components, rather than separate files (i.e. the full filename is `Icons.tsx`). The reason for that is that that allows us to inline the SVGs, which then allows us to use `currentcolor`, which is useful for symbolic items that have just a single colour that might differ depending on their context/hover state.